hist/hist/inc/HFitInterface.h

```diff
@@ -50,6 +50,13 @@ namespace ROOT {
          kGraph
       };
 
+      enum EChisquareType {
```

Comment:
```suggestion
      enum class EChisquareType {
```

According to this guideline:
http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Renum-class

You will have to change things like `ROOT::Fit::kNeyman` to `ROOT::Fit::EChisquareType::kNeyman` after this change, but I think it's worth it because using `enum class` is better for type safety.

Reply:
I agree this is better since the enum is defined outside a class